src/ajax.js

```diff
@@ -18,7 +18,8 @@ define( [
 var
 	r20 = /%20/g,
 	rhash = /#.*$/,
-	rts = /([?&])_=[^&]*/,
+	rantiCache = /(?:[?&])_=[^&]*&?/,
+	remptyQuery = /\?$/,
```

Comment:
I wouldn't worry about this at all, since `/path?&_=1` is valid. But if we do care, then it can be solved by restricting the existing regex capture group rather than introducing a new regex: `/(?:(\?)|&)_=[^&]*/` or (probably smaller) `/(\?)_=[^&]*|&_=[^&]*/`. It should be a separate PR, though.